lib/index.js

```diff
@@ -249,10 +249,17 @@ function promisifyAll(obj) {
  * promise is rejected with that collection. Original indices of the list
  * are preserved.
  *
+ * If the `settle` property in the `options` parameter is set, the returned
```

Comment:
You should probably make this say "if the `sparse` property" not the settle property.